bundles/org.openhab.binding.opensprinkler/src/main/java/org/openhab/binding/opensprinkler/internal/model/NoCurrentDrawSensor.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.opensprinkler.internal.model;
+
+/**
+ * Indicates, that a device is missing a sensor to measure the current draw of itself.
+ *
+ * @author Florian Schmidt - Initial contribution
+ */
+public class NoCurrentDrawSensor extends Exception {
```

Comment:
Exceptions should have "Exception" as their suffix, but I also think that you could give this a more generic name, like "MissingSensorException".

Reply:
Why should it have "Exception" as a suffix. The name already indicates, that's something exceptional, and it extends Exception, so it's pretty clear, that it's an exception, sin't it? What else should it be? However, if you say so, I'll add exception at the end.

Why should the name more generic? The only usage of the exception is, when no sensor for measuring the current draw is present. If it's more generic, the exception probably doesn't transport the intention anymore, isn't it? :/

Reply:
Well the xxxException is the naming convention that we follow. Also I suggested that it could be more generic but `NoCurrentDrawSensorException` works just fine as well. I was only suggesting something more generic in case there are other sensors that may also be detected missing so that they can reuse that custom exception.

But once again all I want is to have the name changed to match our naming conventions.

Reply:
Understood :) The `Exception` prefix is added with the last commit. I would change the name to a more generic exception, once we need to have a more generic one. At the moment, there is no other sensor we might miss :)